samples/State-Management-Bot/main.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""
+This sample shows how to create a simple EchoBot with state.
+"""
+import os
+import yaml
+
+from aiohttp import web
+from botbuilder.schema import Activity, ActivityTypes
+from botbuilder.core import (BotFrameworkAdapter, BotFrameworkAdapterSettings, TurnContext,
+                             ConversationState, MemoryStorage, UserState, CardFactory)
+
+from bots import StateManagementBot
+
+relative_path = os.path.abspath(os.path.dirname(__file__))
+path = os.path.join(relative_path, "config.yaml")
+with open(path, 'r') as ymlfile:
+    cfg = yaml.safe_load(ymlfile)
+
+APP_ID = ''
+APP_PASSWORD = ''
```

Comment:
Just a sample, but any reason why port coming from cfg and these aren't?  Just curious.